windowsforms/Sudoku/VB/sudoku/gui/Form1.vb

```diff
@@ -0,0 +1,83 @@
+﻿Public Class Form1
+
+    Private WithEvents Game As New Game
+    Private ReadOnly _r As New Random
+
+#If NETCOREAPP Then
```

Comment:
I'm curious why this conditional is here?

Reply:
Some day this code will not be needed when VB fully supports Core, until then it is need so this is an easy way to find all the places that need to be fixed. If is also not needed for Framework.

Reply:
@Thraka I have used two other approaches to address the missing code. Create a new file Program.vb and put the code in there or try to replicate the missing VB code and create Application.Designer.vb and put the #IF there. I am open to either solution. I don't expect this will be fixed until Core 5. The code works with Framework and Core with the #IF.

Reply:
I followed the C# example and created a Program.vb Module and moved the conditional code there and removed the condition.